docs/reference/coding-conventions.md

```diff
@@ -18,10 +18,17 @@ If in doubt default to the Java Coding Conventions such as:
 * use 4 space indentation
 * public functions should have documentation such that it appears in Kotlin Doc
 
-Kotlin does not have fields as a primary concept in the language -- it only has properties.
-Avoid the use of prefixes on properties, such as _ or m_ or other kinds of notation.
-If you need access to a backing field of a property, use the $ prefix: `$foo` to refer to a field behind property `foo`;
-never create a private property and call it `_foo`
+In some cases, it's necessary to define a [backing property](properties.html#backing-properties) for a property.
+(For example, it can be necessary when the public view of a property is a read-only collection, and inside the class
+the collection is mutable). In that case, the name of the backing property should use the _ prefix:
```

Comment:
Personally I don't think this should be in the official style guide. I'm not sure we even use this exact scheme in our project. Why prefix and not postfix, for example? Also it's located way too high for its relevance